Frameworks/UIKit/UIButton.mm

```diff
@@ -184,7 +184,7 @@ - (void)_initUIButton {
     _contentVerticalAlignment = UIControlContentVerticalAlignmentCenter;
     _contentHorizontalAlignment = UIControlContentHorizontalAlignmentCenter;
 
-    __block UIButton* weakSelf = self;
+    __weak UIButton* weakSelf = self;
```

Comment:
interesting, block will +1 for reference count in arc but not for non-arc case.

---
In reply to: [98295339](https://github.com/Microsoft/WinObjC/pull/1861#discussion_r98295339) [](ancestors = 98295339,98295161)